README.md

```diff
@@ -1,16 +1,20 @@
+# Airflow [Parcel](https://github.com/cloudera/cm_ext/wiki/Parcels:-What-and-Why%3F)
 
+This repository allows you to install [Apache Airflow](https://airflow.apache.org/) as a parcel deployable by [Cloudera Manager](https://www.cloudera.com/products/product-components/cloudera-manager.html).
 
-## Installing the Parcels
-1. In the Cloudera Manager, go to `Hosts -> Parcels -> Configurations`.
-2. Add `http://teamclairvoyant.s3-website-us-west-2.amazonaws.com/apache-airflow/cloudera/parcels/` to the Remote Parcel Repository URLs.
-3. Airflow parcels and their respective verisons will be availble within the parcels page. 
-4. Download, Distribute, Activate the required parcels to use them. 
+## Installing the Parcel
+0. First, install the [Airflow CSD](https://github.com/teamclairvoyant/apache-airflow-cloudera-csd).  Then you can skip steps #1 and #2.
```

Comment:
Wouldn't users still have to execute Steps 1 and 2 after installing the CSD?

Reply:
The CSD ships with the parcel location (see [PR #2](https://github.com/teamclairvoyant/apache-airflow-cloudera-csd/pull/2/files)), so they can be skipped.